src/test/rustdoc-js-std/parser-invalid.js

```diff
@@ -0,0 +1,64 @@
+// This test is mostly to check that the parser still kinda outputs something
+// (and doesn't enter an infinite loop!) even though the query is completely
+// invalid.
+const QUERY = ['a b', 'a   b', 'a,b(c)'];
+
+const PARSED = [
+    {
+        elems: [
+            {
+                name: "a",
+                fullPath: ["a"],
+                pathWithoutLast: [],
+                pathLast: "a",
+                generics: [],
+            },
+            {
+                name: "b",
+                fullPath: ["b"],
+                pathWithoutLast: [],
+                pathLast: "b",
+                generics: [],
+            },
+        ],
+        foundElems: 2,
+        original: "a b",
+        returned: [],
+        typeFilter: -1,
+        userQuery: "a b",
+        error: null,
```

Comment:
It's weird that this one is in the `parser-invalid` test case, but is not actually reporting an error.

No part of the eBNF describes this syntax, so presumably it's invalid, and there's just an error-handling bug here?

Reply:
There is no syntax error so to speak. So it seems normal to have this as result. The `parser-invalid` name came from cases I found that were triggering parser bugs (as the top comment explains it). However, the file name is definitely not be the best... So do you want me to rename the file or do you have something else in mind?

Reply:
I would rename the file, if this is how it's going to work, but the really important part is that the eBNF be changed to accept this syntax. It seems to reject it right now.

Reply:
I'll make the comma optional.